src/python/dbs/apis/dbsClient.py

```diff
@@ -760,7 +776,7 @@ def listBlockTrio(self, **kwargs):
         requiredParameters = {'forced': ['block_name']}
         checkInputParameter(method="listBlockTrio", parameters=list(kwargs.keys()), validParameters=validParameters,
                             requiredParameters=requiredParameters)
-        return self.__callServer("blockTrio", params=kwargs, callmethod='GET')
+        return self.__callServer("blockTrio", params=kwargs, callmethod='GET', aggFunc=aggBlockTrio))
```

Comment:
There is a close parenthesis in excess here.

Reply:
ok, my typo